lib/Dialect/Calyx/CalyxOps.cpp

```diff
@@ -182,6 +182,28 @@ LogicalResult calyx::verifyControlLikeOp(Operation *op) {
   }
   return verifyControlBody(op);
 }
+Operation *calyx::getExternHWModule(OpBuilder &builder, ComponentOp op) {
```

Comment:
Should it return HWModuleExtern, to make it easier for users to play with? These can be implicitly casted back to `Operation *`, so I don't think you are losing anything.

Reply:
Agreed.